platform/spire/src/seq.py

```diff
@@ -129,4 +134,5 @@ def configure(command: list, parser: argparse.ArgumentParser):
     "supervisor": wrapseq("set up and verify functionality of entire supervisor node (keysystem + ssh)",
                                sequence_supervisor),
     "cluster": wrapseq("set up and verify kubernetes infrastructure operation", sequence_cluster),
+    "redeploy": wrapseq("redeploy a cluster configuration to a running cluster", sequence_redeploy_config),
```

Comment:
needs rebase

Reply:
I have rebased this branch and have adjusted the changes to fit the new version of spire. I have also tested the changes to ensure that they still work.